server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -238,6 +240,10 @@ public void apply(Settings value, Settings current, Settings previous) {
     public static Set<Setting<?>> BUILT_IN_CLUSTER_SETTINGS = Collections.unmodifiableSet(
         new HashSet<>(
             Arrays.asList(
+                AdmissionControlSettings.ADMISSION_CONTROL_TRANSPORT_LAYER_MODE,
```

Comment:
Lets add it near the end or next sb settings and add a comment that these are admission control settings.